lib/Thepeer.js

```diff
@@ -0,0 +1,154 @@
+    } catch (e) {
+      Helper.processError(e)
+    }
+  }
+
+  /**
+   * @param {string} reference - The reference returned when the user was indexed
+   * @param {string} identifier - The identifier of the user
+   * @returns {JSON}  A JSON response containing the details of the user
+   * @memberof ThePeer
+   */
+  async updateUser (reference, identifier) {
+    try {
+      const response = await this.request.put(`/users/${reference}`, {
+        identifier: identifier
+      })
+      return response.data
+    } catch (e) {
+      Helper.processError(e)
+    }
+  }
+
+  /**
+   * @param {string} reference - The reference returned when the user was indexed
+   * @returns {JSON}  A JSON response containing the details of the user
+   * @memberof ThePeer
+   */
+  async deleteUser (reference) {
+    try {
+      const response = await this.request.delete(`/users/${reference}`)
+      return response.data
+    } catch (e) {
+      Helper.processError(e)
+    }
+  }
+
+  /**
+   * @param {string} reference - The reference returned when the user was indexed
+   * @returns {JSON}  A JSON response containing the details of the user
+   * @memberof ThePeer
+   */
+  async getUser (reference) {
+    try {
+      const response = await this.request.get(`/users/${reference}`)
+      return response.data
+    } catch (e) {
+      Helper.processError(e)
+    }
+  }
+
+  /**
+   *
+   * @param {string} linkid - The id of a user linked account
+   * @returns {JSON} A JSON response containing the details of the user
+   * @memberof ThePeer
+   */
+  async getLink (linkid) {
+    try {
+      const response = await this.request.get(`/link/${linkid}`)
+      return response.data
+    } catch (e) {
+      Helper.processError(e)
+    }
+  }
+
+  /**
+   * A function that charges your user's linked account at any time
+   * @param {string} linkid - The id of the link
+   * @param {integer} amount - amount in kobo
+   * @param {string} remark - The reason for initiating a direct charge
+   * @returns {JSON} A JSON response containing the details of the user
+   * @memberof ThePeer
+   */
+  async chargeLink (linkid, amount, remark) {
+    try {
+      const response = await this.request.post(`/link/${linkid}/charge`, {
+        amount: amount,
+        remark: remark
+      })
+      return response.data
+    } catch (e) {
+      Helper.processError(e)
+    }
+  }
+
+  /**
+   * Authorize direct charge request via webhooks
+   * @param {*} reference - The direct charge reference sent via webhook
+   * @param {boolean} insufficientFunds - Status of user funds
+   * @returns {JSON} A JSON response containing the details of the user
+   * @memberof ThePeer
+   */
+  async authorizeDirectCharge (reference, insufficientFunds) {
+    try {
+      const response = await this.request.post(`/debit/${reference}`, {
+        insufficient_funds: insufficientFunds
+      })
+      return response.data
+    } catch (e) {
+      Helper.processError(e)
+    }
+  }
+}
```

Comment:
leave a new line before `module.exports`

Reply:
Skipping the todo was an oversight. I'll work on that ASAP. Also do you by chance have a sandbox secret key so I could write some test coverage.